public/locales/en/files.json

```diff
@@ -0,0 +1,62 @@
+{
+  "title": "Files",
+  "home": "Home",
+  "breadcrumbs": "Breadcrumbs",
+  "finished": "Finished!",
+  "totalSize": "Total size: {size}",
+  "filesSelected": "{count, plural, one {File selected} other {Files selected}}",
+  "individualFilesOnly": "Only available for individual files",
+  "fileName": "File name",
+  "size": "Size",
+  "noPDFSupport": "Your browser does not support PDFs. Please download the PDF to view it:",
+  "downloadPDF": "Download PDF.",
+  "downloadInstead": "Try <1>downloading</1> it instead.",
+  "cantBePreviewed": "Sorry, this file can't be previewed",
+  "addFile": "Add file",
+  "addFolder": "Add folder",
+  "addByPath": "Add by path",
+  "addToIPFS": "Add to IPFS",
+  "newFolder": "New folder",
+  "actions": {
+    "add": "Add",
+    "delete": "Delete",
+    "rename": "Rename",
+    "download": "Download",
+    "inspect": "Inspect",
+    "copyHash": "Copy Hash",
+    "share": "Share",
+    "close": "Close",
+    "copy": "Copy",
+    "cancel": "Cancel",
+    "clear": "Clear",
+    "create": "Create",
+    "save": "Save",
+    "unselectAll": "Unselect All"
+  },
+  "shareModal": {
+    "title": "Share Files",
+    "description": "Copy the link below and share it with your friends."
+  },
+  "renameModal": {
+    "titleFile": "Rename File",
+    "titleFolder": "Rename Folder",
+    "descriptionFile": "Choose a new name for this file.",
+    "descriptionFolder": "Choose a new name for this folder."
+  },
+  "deleteModal": {
+    "titleItem": "{count, plural, one {Delete Item} other {Delete {count} Items}}",
```

Comment:
FYI I had to tweak the ICU format used for plurals because Transifex supports only a subset of ICU when JSON files are used (I think the full support is in gettext):

- https://docs.transifex.com/formats/json#plurals-support

Anyway, it now renders correctly. Below is an  example in Polish, where  we have more than two plural forms :upside_down_face: , and the UI at Transifex now displays them all: 

> ![screenshot_18](https://user-images.githubusercontent.com/157609/45130358-92b7fc80-b188-11e8-8d65-39d169e4938b.png)